zsl/constants.py

```diff
@@ -0,0 +1,8 @@
+class MimeType(object):
+    APPLICATION_JSON = 'application/json'
```

Comment:
There's no problem in length, you've content assists unless you have vims and 80 chars per line. There's the problem in the fact, that if you'd like to be web independent, then you'll have to use your own types in the future and we do want to be. 

Then using this constants can at least point you to the relevant places when refactoring. And there's a good practice to use constants. Content-type Content-Type or content-type should I use? No HttpHeaders.CONTENT_TYPE and the game is over.